### DISCUSSION TITLE
Manual Querying in V2

### DISCUSSION THREAD

**Discussion by Xadeka** [TanStack/query#606] [General] [answered]
In v1, there was the `manual` config option that allowed a query to only be fetched if `refetch` was called. I also see "Manual Querying" in the readme's table of contents but the section no longer exists.

Is a similar feature to this `manual` option planned to be added? Or possibly a better question is, is there a recommended alternative to this config option that would make the hooks work similarly to a "manual" query in v1?

For reference, this is the Manual Querying section from 1.5.9:
https://github.com/tannerlinsley/react-query/tree/v1.5.9#manual-querying

**Comment by Xadeka** [comment, marked as answer]
Looks like the `enabled` config option covers what I'm looking for. Setting `enabled` to `false` allows me to call `refetch` to manually fetch the data.

https://github.com/tannerlinsley/react-query#disabling-or-pausing-a-query

**Reply by rickvian** [reply]
> Looks like the `enabled` config option covers what I'm looking for. Setting `enabled` to `false` allows me to call `refetch` to manually fetch the data.
> 
> https://github.com/tannerlinsley/react-query#disabling-or-pausing-a-query

This is the answer i looking for, i was searching for `useQuery fetch only once, fetch one time`

**Comment by grantweii** [comment]
Previously, manual also prevented the query from fetching when its dependencies were changed. I believe the enabled option does not cover this scenario?

**Reply by ayepRahman** [reply]
facing the same issue

**Reply by TkDodo** [reply]
> Previously, manual also prevented the query from fetching when its dependencies were changed. I believe the enabled option does not cover this scenario?

not entirely sure what that means, but as long as a query is disabled, it should not fetch. so yeah I think the enabled option covers that.
